db/schema.rb

```diff
@@ -39,6 +39,16 @@
     t.index ["singleton_guard"], name: "index_myconfigs_on_singleton_guard", unique: true
   end
 
+  create_table "offsprings", force: :cascade do |t|
+    t.string   "first_name", null: false
+    t.string   "last_name", null: false
+    t.string   "type", null: false
```

Comment:
No, Rails fills it... You don't want to mess with Rails updating it. We don't control that so we don't mess with it